tools/pipelines/templates/build-npm-package.yml

```diff
@@ -314,6 +314,7 @@ stages:
                   command: 'custom'
                   workingDir: ${{ parameters.buildDirectory }}
                   customCommand: 'run ${{ taskTestStep }}:coverage'
+                  retryCount: '1'
```

Comment:
i'm pretty strongly against doing something like this. in my experience it leads to less reliability over all, as it allows new flaky tests to be added. Overtime the flakiness adds up, and then the build starts failing again. this is then a really hard situation to get out of, as now you have a bunch of flaky tests. 

our current pattern of skipping flaky tests keeps does a good job of not allowing new flaky tests, and keeps the reliably high for our non skipped tests.

Reply:
I think that's a good point and I agree. I will close the PR and the task which were intended to add a retry of 1 for the test failures. As you mentioned, I also agree that the pros of exposing the potential flaky tests should be greater than the pros of handling sporadic test failures. Thank you.